xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+         look.
+      </para>
+      <example xml:id="ex-ha-qdevice-crm-corosync-status-qnetd">
+         <title>Output Status of &qnet; Server</title>
+         <screen>&prompt.root;<command>crm corosync status quorum</command> <co xml:id="co-quorum-qnet-cmd"/>
+Cluster "hacluster": <co xml:id="co-quorum-qnet-name"/>
+    Algorithm:          Fifty-Fifty split <co xml:id="co-quorum-qnet-algo"/>
+    Tie-breaker:        Node with lowest node ID
+    Node ID 3232235777: <co xml:id="co-quorum-qnet-alice"/>
+        Client address:         ::ffff:&subnetI;.1:54732
+        HB interval:            8000ms
+        Configured node list:   3232235777, 3232235778
+        Ring ID:                aa10ab0.8
+        Membership node list:   3232235777, 3232235778
+        Heuristics:             Undefined (membership: Undefined, regular: Undefined)
+        TLS active:             Yes (client certificate verified)
+        Vote:                   ACK (ACK)
+    Node ID 3232235778:
+        Client address:         ::ffff:&subnetI;.2:43016
+        HB interval:            8000ms
+        Configured node list:   3232235777, 3232235778
+        Ring ID:                aa10ab0.8
+        Membership node list:   3232235777, 3232235778
+        Heuristics:             Undefined (membership: Undefined, regular: Undefined)
+        TLS active:             Yes (client certificate verified)
+        Vote:                   No change (ACK)</screen>
+       <calloutlist>
+          <callout arearefs="co-quorum-qnet-cmd">
+             <para>
+                You can also use the command <command>corosync-qnetd-tool</command>
+                as an alias with an identical result.
+             </para>
+          </callout>
+          <callout arearefs="co-quorum-qnet-name">
+             <para>
+                The name of your cluster as set in the configuration file
+                <filename>/etc/corosync/corosync.conf</filename> in the
+                <literal>totem.cluster_name</literal> section.
+             </para>
+          </callout>
+          <callout arearefs="co-quorum-qnet-algo">
+             <para>
+                The algorithm currently used, here the FFSplit.
+             </para>
+          </callout>
+          <callout arearefs="co-quorum-qnet-alice">
+             <para>
+                This is the entry for the node witht he IP address
+                <systemitem class="ipaddress">&subnetI;.1</systemitem>.
+                TLS is active.
+             </para>
+          </callout>
+       </calloutlist>
+      </example>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-more">
+      <title>For More Information</title>
+      <itemizedlist>
+         <listitem>
+            <para>Man pages of corosync-qdevice(8), corosync-qnetd(8), </para>
+         </listitem>
+         <listitem>
+            <para><remark>toms 2020-05-12: any further sources which could be valuable?</remark></para>
```

Comment:
This is not something for our users :-),  but it could be another reference internally for ourselves:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_high_availability_clusters/assembly_configuring-cluster-quorum-configuring-and-managing-high-availability-clusters#assembly_configuring-quorum-devices-configuring-cluster-quorum